### ISSUE TITLE
dataset=True and sanitize_columns=False not working

### ISSUE THREAD

**Issue by chrisrana** [awslabs/aws-data-wrangler#380]
*P.S. Don't attach files. Please, prefer add code snippets directly in the message body.*
I am trying to write parquet in a folder and i am  using below code...it is sanitizing the column name to smaller case and 
SetID_OBR becomes set_id_obr

wr.s3.to_parquet(
                partition_cols=partition_col,
                df=df,
                path=s3_write_path+"/",
                sanitize_columns=False,
                dataset=True

            )

As per docs if dataset=True sanitize_columns will always be true...
is there any way to make sanitize_columns=False forcefully...?

**Comment by chrisrana** [comment]
I understand that sanitize_columns is forced in case of dataset=True to ensure Athena compatibility...what if someone dont want to sanitize columns name...I want to store parquet and query through Spark...in that case it doesn't matter column name is sanitized or not....

**Comment by igorborgest** [comment]
Hi @chrisrana 

It is a fair request indeed.

I've just made the related changes in the commit above, and now the sanitisation is forced only when there is some Glue catalog integration.

Do you mind to test it from our development branch and check if it helps in your use case?
The ideia is to publish this enhancement in the version `1.9.1` later today.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released on version **[1.9.1](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.9.1)**

**Comment by chrisrana** [comment]
Tested it it works fine.Thanks @igorborgest
